jx_sqlite/expressions.py

```diff
@@ -29,7 +29,7 @@
 
 @extend(Variable)
 def to_sql(self, schema, not_null=False, boolean=False):
-    cols = [c for cname, cs in schema.items() if startswith_field(cname, self.var) for c in cs]
+    cols = [c for cname, cs in schema.map_to_sql().items() if startswith_field(cname, self.var) for c in cs]
```

Comment:
using `map_to_sql()` works good except it breaks few other tests like test_select_gt_on_sub. Working  to solve the issue.